portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -71,9 +71,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     /** 
-     * Stores the comment into datastore and redirect user back to home page 
-     * after submission. datastore.put() directs to /data page so the redirect
-     * prevents the user from reaching the /data page.
+     * Stores the comment into Datastore and redirect user back to home page 
+     * after submission. The form to submit the comment on the website directs 
+     * to /data page so the redirect prevents the user from reaching the /data page.
```

Comment:
```suggestion
     * to /data page so the redirect prevents the user from getting stuck on the empty /data page.
```

Also, please reflow the text.